### PR TITLE
Create portfolio page

### PR DESCRIPTION
**Pull request by vaishnavi-janardhan**
- Add brief intro section
- Add list of projects
- Add journey section
- Add social footer

### Preview of the webpage
preview1
preview2

### CODE REVIEW

**Comment by skashish-sps** [comment]
Hi,
It will be great if you can deploy your site, and share links, adding a full page screenshot is good, but nothing beats the actual look and feel to it

File: portfolio/src/main/webapp/lib/typed.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*!
+ * 
+ *   typed.js - A JavaScript Typing Animation Library
```
Comment (skashish-sps) [inline portfolio/src/main/webapp/lib/typed.js]: Prefer to use CDN's, reduces the code duplication and maintenance effort

Comment (skashish-sps) [reply portfolio/src/main/webapp/lib/typed.js]: <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -1,28 +1,11 @@
...
+  var options = {
+    strings: ['More updates coming soon', 'Stay home, stay safe', 'Code, Coffee, Sleep, Repeat'],
+    loop: true,
+    startDelay: 1000,
+    typeSpeed: 50,
+    backDelay: 1000,
+    backSpeed: 50
+  };
+  var typed = new Typed('.typed', options);
+});
```
Comment (skashish-sps) [inline portfolio/src/main/webapp/script.js]: Use new line at end of all files as a convention

**Review by skashish-sps — COMMENTED** [review]
Prefer to use CDN's, reduces the code duplication and maintenance effort
Typed JS : <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>

Do you own the images or are they downloaded, if they are downloaded then prefer to reference them from the network rather than your own repository.

Also, at times leads to issues, with copyright

File: portfolio/src/main/webapp/lib/typed.js (hunk 1 above)
Comment (vaishnavi-janardhan) [reply portfolio/src/main/webapp/lib/typed.js]: Removed the script and added link to the CDN.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (vaishnavi-janardhan) [reply portfolio/src/main/webapp/script.js]: Okay.

**Comment by vaishnavi-janardhan** [comment]
> Do you own the images or are they downloaded, if they are downloaded then prefer to reference them from the network rather than your own repository.
> 
> Also, at times leads to issues, with copyright

The project images are my own.
I generated the other 2 avatar images using GBoard Minis, hence I can't reference them using links.

**Comment by vaishnavi-janardhan** [comment]
I have deployed the site
https://vjanardhan-sps-summer20.appspot.com/

**Review by skashish-sps — APPROVED** [review]
LGTM

**Pull request merged**